src/Symfony/Component/Encryption/EncryptionInterface.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption;
+
+use Symfony\Component\Encryption\Exception\DecryptionException;
+use Symfony\Component\Encryption\Exception\EncryptionException;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+interface EncryptionInterface
+{
+    /**
+     * Generates new a Key to be used with encryption.
```

Comment:
```suggestion
     * Generates new a KeyInterface to be used with encryption.
```

Reply:
I think this is correct. We are generating a "new Key" or a "new object implementing KeyInterface". But we are not generating "a new KeyInterface"

Reply:
It should be `key` with a lower-case K then. 🙃

Reply:
Okey. 

Thank you